contents.json

```diff
@@ -1425,6 +1431,18 @@
 		"homepage": "https://github.com/novi/mysql-swift",
 		"tags": ["linux"]
 	}, {
+		"title": "Perfect-MySQL",
+		"category": "sql-drivers",
+		"description": "A stand-alone Swift wrapper around the MySQL client library, enabling access to MySQL servers.",
+		"homepage": "https://github.com/PerfectlySoft/Perfect-MySQL",
+		"tags": ["linux", "macOS"]
+	}, {
+		"title": "Perfect-MariaDB",
```

Comment:
please remove this project as it does not meet the minimum number of github stars (15)